include/seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp

```diff
@@ -204,7 +213,9 @@ template <sequence_container_concept seq_type>
 //!\cond
     requires std::Assignable<reference_t<seq_type>, gap const &>
 //!\endcond
-inline typename seq_type::iterator insert_gap(seq_type & seq, typename seq_type::const_iterator pos_it, typename seq_type::size_type count)
+inline typename seq_type::iterator insert_gap(seq_type & seq,
+                                              typename seq_type::const_iterator pos_it,
+                                              typename seq_type::size_type count)
```

Comment:
call it size maybe instead of count? The semantic would be to insert a single gap of a certain size and not count many gap characters, right?

Reply:
Yes, I agree.